contrib/epee/include/net/abstract_tcp_server2.inl

```diff
@@ -146,7 +146,7 @@ PRAGMA_WARNING_DISABLE_VS(4355)
 
     context = boost::value_initialized<t_connection_context>();
     const unsigned long ip_{boost::asio::detail::socket_ops::host_to_network_long(remote_ep.address().to_v4().to_ulong())};
-    m_local = epee::net_utils::is_ip_loopback(ip_);
+    m_local = epee::net_utils::is_ip_local(ip_);
```

Comment:
What does this change entail? Shouldn't this be a separate commit for better visibility?

Reply:
It allows non routable IPs to use the longer timeout. It could be a separate commit, I just didn't see enough of a need since it's related to the same thing, but it's true it's a separate unit.